src/index.html

```diff
@@ -5,8 +5,403 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header_logo-block">
+        <img 
+          src="./images/logo.png" 
+          alt="logo" 
+          class="header_logo-block_img">
+      </div>
+      <nav class="header_nav">
+        <ul class="header_nav_list">
+          <li class="header_nav_list_link">
+            <a href="APPLE" class="is-active">APPLE</a>
+          </li>
+          <li class="header_nav_list_link">
+            <a href="SAMSUNG" class="header_nav_list_link_text">SAMSUNG</a>
+          </li>
```

Comment:
```suggestion
        <ul class="header_nav_list">
          <li class="header_nav_list_link">
            <a href="APPLE" class="is-active">APPLE</a>
          </li>
          
          <li class="header_nav_list_link">
            <a href="SAMSUNG" class="header_nav_list_link_text">SAMSUNG</a>
          </li>
          
```
please fix code style all over your code